openeo/extra/job_management.py

```diff
@@ -253,8 +258,11 @@ def _make_resilient(connection):
         connection.session.mount("https://", HTTPAdapter(max_retries=retries))
         connection.session.mount("http://", HTTPAdapter(max_retries=retries))
 
-    def _normalize_df(self, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
users are apparently using this private method `_normalize_df`, so we might want to be careful to remove/change it